itest/psbt_test.go

```diff
@@ -3619,10 +3619,19 @@ func sendToTapscriptAddr(ctx context.Context, t *harnessTest, alice,
 	AssertSendEventsComplete(t.t, bobAddr.ScriptKey, sendEvents)
 }
 
+// sendAssetAndAssertResponse is a struct that holds the response of a
+// sendAssetsToAddr call.
+type sendAssetAndAssertResponse struct {
+	// RpcResp is the response message returned by the SendAsset RPC
+	// endpoint.
+	RpcResp *taprpc.SendAssetResponse
```

Comment:
Why do we need this wrapper around the existing type? Can't we just return the RPC response directly?